appinventor/appengine/src/com/google/appinventor/client/editor/simple/SimpleComponentDatabase.java

```diff
@@ -61,7 +63,10 @@ public static SimpleComponentDatabase getInstance() {
   private static final ComponentResource componentResources = GWT.create(ComponentResource.class);
 
   private SimpleComponentDatabase() {
-    super(new ClientJsonParser().parse(
-        componentResources.getSimpleComponents().getText()).asArray());
+    super(new ClientJsonParser().parse(componentResources.getSimpleComponents().getText())
```

Comment:
Not important, just gets things to compile since I changes simple_components.json